api/capi/include/nntrainer.h

```diff
@@ -507,6 +507,55 @@ int ml_train_dataset_set_property_for_usage(ml_train_dataset_h dataset,
                                             ml_train_dataset_data_usage_e usage,
                                             ...);
 
+/**
+ * @brief Save the model
+ * @details Use this function to save the current model. @a format
+ * describes various formats in which various selections of the
+ * parameters of the models can be saved. Some formats may save
+ * parameters required for training. Some other formats may save model
+ * configurations. Unless stated otherwise, @a ml_train_model_compile() has to
+ * be called upon the @a model before calling this function.
+ * @see @a ml_train_model_format_e to check which part of the model is
+ * saved
```

Comment:
missing `PERIOD` at the end of the statement.

Reply:
Done